app/components/EntryReference.tsx

```diff
@@ -0,0 +1,34 @@
+import type { ReferenceLink } from "@prisma/client"
+import SanitizedTextSpan from "./SanitizedTextSpan"
+
+type EntryReferenceProps = {
+  referenceLink: ReferenceLink
+}
+
+export default function EntryReference({
+  referenceLink,
+}: EntryReferenceProps): JSX.Element {
+  console.log("referenceLink", referenceLink)
+  return (
+    <li>
+      {/*  For some reason Prisma does not properly generate the 'reference'
+      reference! (Maybe due to the name?) It does exist, though.
+      @ts-ignore */}
```

Comment:
that's so weird haha, did you restart vscode? cause I've had the same problem go away when I do that. Apart from that prisma can be a bit finicky

Reply:
I did! I restarted VS Code, did `prisma regenerate` multiple times, removed `node_modules`, etc... and this error persists when running `npm run typecheck` from the command line, too, which makes me think it's truly a Prisma error, not a VS Code one.